src/tests/ftest/server/daos_server_config.yaml

```diff
@@ -167,5 +167,5 @@ server_config_val: !mux
   targets_str:
     config_val:
       - "targets"
-      - "ABDCD"
+      - -1
```

Comment:
We want both fail test cases, `-1` and `ABDCD`, so this should be a complete new entry.

Reply:
This change is already done with the latest patch and is undergoing CI testing.